### PR TITLE
VisAvail update

### PR DESCRIPTION
**Pull request by IanMayo**
This is a tracking PR, to allow ongoing review (& preview via Heroku) of the updates to the client-side components.

### CODE REVIEW

**Comment by IanMayo** [comment]
Hello @tanganellilore  - I've just invited the analysts to review the Heroku [deployment](https://pepys-timeli-lorenzo-de-ewozhs.herokuapp.com/)

They have this feedback, which seems quite sensible:
* [x] The platform names would be more legible if the font-size was increased to almost the height of the bar.
* [x] The %age values on the right-hand side are hard to read. If we move them outside the bar (to the right) they will be easier to read
* [x] We may have to fit up to 25 ships for a Serial.  Could we reduce the vertical whitespace between the timeline bars please?  One pixel spacing would be a good first step.
* [x] Per-serial %age coverage.  Five serials are shown in the mockup.  A serial with high overall coverage may almost be ready for analysis.  A serial with low coverage may require chasing up the data.  If we shaded the background of the serial titles with red/amber/green it would help this understanding.

**Comment by tanganellilore** [comment]
So, first 3 point should be addressed in easy way but for the 4th, just to understand better:

Is it required a background color on the title of serial name in base of total %age?
Like picture below?
image

If yes, how you calculate it? It's passed by query, so I can just add a new keyword on dataset like "overall_averege" and it will assign a class base of this value, or is a median of all percentage in a series?

**Comment by IanMayo** [comment]
Yes - it's acceptable to pass the `overall_average` as a new parameter.

**Comment by tanganellilore** [comment]
4 points closed, I hope.
If you need letter/bar more bigger, tell me.

Function updated for catch overall_percentage and adapt new css class

File: css/style.css [hunk 1]
```diff
@@ -57,49 +60,66 @@ h1,h2,h3,h4,h5 {
     fill: #f5c84266 !important;
 }
 .visavail .ypercentage {
-    paint-order: stroke;
+    /* paint-order: stroke;
```
Comment (IanMayo) [inline css/style.css]: @tanganellilore - if this block is commented out, should we just delete it?

Comment (tanganellilore) [reply css/style.css]: yes, class can be remove if actual version is ok.
Also on index.html is present a commented old code, but I prefer, before delete some code,  maintain commented if some rollback will be required

Comment (tanganellilore) [reply css/style.css]: When everything will be approved, I proceed with deletion of comment line

**Comment by IanMayo** [comment]
It's all great, thanks @tanganellilore .

**Comment by tanganellilore** [comment]
So i can upload version without comment and clean code?

One more thing, I suggest you tu add JQuery if you have in mind to retrive something from backend with/without human interaction and without usage of react/angular.

**Comment by IanMayo** [comment]
I'm happy to keep comments, but yes - please upload a version with clean code.

Thanks for the advice on JQuery.  We have a working version of the page that retrieves data without introducing a JQuery dependency:
https://github.com/debrief/pepys-timeline/pull/6/files#diff-1e015626c0dbe5879bf395d136c1c4c82fd93e49fcf771693fbd366cc87dd7f8

That code will get updated to use your work in this branch.

**Comment by IanMayo** [comment]
@tanganellilore - is your work now complete in this branch?

**Comment by tanganellilore** [comment]
for my point of view, yes.
I have added one folder for image, so you can upload image on this and everything is offline mode.

Thanks

**Pull request merged**